rpc/core/types/responses.go

```diff
@@ -54,14 +54,19 @@ func NewResultCommit(header *types.Header, commit *types.Commit,
 	}
 }
 
+type ValidatorStatus struct {
+	VotingPower int64 `json:"voting_power"`
+}
+
 type ResultStatus struct {
-	NodeInfo          p2p.NodeInfo  `json:"node_info"`
-	PubKey            crypto.PubKey `json:"pub_key"`
-	LatestBlockHash   cmn.HexBytes  `json:"latest_block_hash"`
-	LatestAppHash     cmn.HexBytes  `json:"latest_app_hash"`
-	LatestBlockHeight int64         `json:"latest_block_height"`
-	LatestBlockTime   time.Time     `json:"latest_block_time"`
-	Syncing           bool          `json:"syncing"`
+	NodeInfo          p2p.NodeInfo    `json:"node_info"`
+	PubKey            crypto.PubKey   `json:"pub_key"`
```

Comment:
shall we move this pubkey under the ValidatorStatus ?

Reply:
Maybe we should change Status to consist of NodeInfo, SyncInfo, ValidatorInfo.

SyncInfo contains all the `LatestXxx` and the `Syncing` bool. `ValidatorInfo` contains the pubkey and the voting power. 

Thoughts ? No need to do it here but let's open a new issue for the idea if you like it

Reply:
What we have right now is good enough. I would not make such changes without explicit feedback from users.

Reply:
I like the idea. The question is whenever our users will like it or not

Reply:
https://github.com/tendermint/tendermint/issues/1417